proto/frequenz/api/common/v1/microgrid/electrical_components/electrical_components.proto

```diff
@@ -121,6 +121,54 @@ enum ElectricalComponentStatus {
   ELECTRICAL_COMPONENT_STATUS_INACTIVE = 2;
 }
 
+// `ElectricalComponentControlMode` explicitly defines whether an electrical
+// component is intended to be active (enabled) or inactive (disabled). This
+// intentional setting directly determines component availability to downstream
+// services and applications.
+//
+// !!! note "Control Mode & Component Availability"
+//     The Control Mode explicitly represents the user's or automation's
+//     intention to make the component either ACTIVE or INACTIVE. This directly
+//     impacts availability:
+//     - ACTIVE indicates the component is intended to be available and usable
+//       by downstream agents and services.
+//     - INACTIVE indicates the component is intended to be unavailable and
+//       ignored by downstream agents and services.
```

Comment:
I would say not usable instead of ignored.

Reply:
I changed it to "unavailable for use", since the component should still be available, but no agent will be able to control it or use it.
Does the phrasing look okay now?

Reply:
yes.